static/app/views/performance/transactionSummary/transactionOverview/content.tsx

```diff
@@ -332,15 +332,22 @@ function SummaryContent({
     handleOpenAllEventsClick: handleAllEventsViewClick,
   };
 
+  const hasTransactionSummaryCleanupFlag = organization.features.includes(
+    'performance-transaction-summary-cleanup'
+  );
+
   return (
     <Fragment>
       <Layout.Main>
-        <FilterActions>
-          <Filter
-            organization={organization}
-            currentFilter={spanOperationBreakdownFilter}
-            onChangeFilter={onChangeFilter}
-          />
+        <FilterActions hasTransactionSummaryCleanupFlag>
```

Comment:
```suggestion
        <FilterActions hasTransactionSummaryCleanupFlag={hasTransactionSummaryCleanupFlag}>
```

Reply:
hmmm I'm of the opinion that passing booleans as props is a lot cleaner if you just attach the prop like I have here

Reply:
but unsure whether this pattern is disliked or not by others

Reply:
nevermind good catch!